src/js/components/utils/detailPageLayout.tsx

```diff
@@ -66,34 +67,38 @@ const DetailPageLayout = ({
           ) : null
         }
       />
-      <div
-        className="slds-p-horizontal_x-large
-          slds-p-top_x-small
-          metecho-breadcrumb
-          slds-truncate"
-      >
-        <BreadCrumb
-          trail={[
-            <Link to={routes.home()} key="home">
-              {i18n.t('Home')}
-            </Link>,
-          ].concat(
-            breadcrumb.map((crumb, idx) => {
-              if (crumb.url) {
+      <div>
+        <div
+          className="slds-p-horizontal_x-large
+            slds-p-top_x-small
+            metecho-breadcrumb
+            slds-is-relative"
+        >
```

Comment:
Just a reminder that this breadcrumb styling is not finalized. I know mobile is a low priority for Metecho, but things get pretty weird on narrow screens...

![Screen Shot 2021-05-24 at 12 06 23 PM](https://user-images.githubusercontent.com/552316/119375573-f829df80-bc88-11eb-8ae8-9f69018633e5.png)

Reply:
And revisit this

Reply:
What we discussed worked for getting the breadcrumb popover to show and to apply the `slds-truncate `styles to the navigation list items. 
![image](https://user-images.githubusercontent.com/41588129/119551366-dbc99800-bd99-11eb-860e-ddc4c16c4ea5.png)

@stacyk @jgerigmeyer 
I think this is looking better. The list items aren't wrapping and the popover for the page title moves to the left on narrow screens.